packages/grant-explorer/.env.sample

```diff
@@ -30,7 +30,7 @@ REACT_APP_ENV=development
 REACT_APP_TAG_MANAGER=XXXXX
 
 # Allo Indexer API Endpoint
-REACT_APP_ALLO_API_ENDPOINT=#########
+REACT_APP_ALLO_API_URL=#########
```

Comment:
Keeping it consistent with the other apps. It's confusing otherwise.
This key is already on fleek but we don't use it